firmware/config/boards/microrusefi/board_configuration.cpp

```diff
@@ -0,0 +1,203 @@
+}
+
+static void setupVbatt()
+{
+	engineConfiguration->vbattDividerCoeff = 8.166666f;
+	engineConfiguration->vbattAdcChannel = EFI_ADC_11;
+
+	// 1k high side/1.5k low side = 1.6667 ratio divider
+	engineConfiguration->analogInputDividerCoefficient = 2.5f / 1.5f;
+
+	engineConfiguration->adcVcc = 3.29f;
+}
+
+static void setupTle8888()
+{
+	// Enable spi3
+	boardConfiguration->is_enabled_spi_3 = true;
+
+	// Wire up spi3
+	boardConfiguration->spi3mosiPin = GPIOB_5;
+	boardConfiguration->spi3misoPin = GPIOB_4;
+	boardConfiguration->spi3sckPin = GPIOB_3;
+
+	// Chip select
+	engineConfiguration->tle8888_cs = GPIOD_5;
+
+	// SPI device
+	engineConfiguration->tle8888spiDevice = SPI_DEVICE_3;
+}
+
+static void setupEtb()
+{
+	// TLE9201 driver
+	// This chip has three control pins:
+	// DIR - sets direction of the motor
+	// PWM - pwm control (enable high, coast low)
+	// DIS - disables motor (enable low)
+
+	// PWM pin
+	boardConfiguration->etb1.controlPin1 = GPIOC_7;
+	// DIR pin
+	boardConfiguration->etb1.directionPin1 = GPIOA_8;
+
+	// Unused
+	boardConfiguration->etb1.controlPin2 = GPIO_UNASSIGNED;
+	boardConfiguration->etb1.directionPin2 = GPIO_UNASSIGNED;
+
+	// we only have pwm/dir, no dira/dirb
+	engineConfiguration->etb1_use_two_wires = false;
+
+	engineConfiguration->etbFreq = 800;
+}
+
+static void setupDefaultSensorInputs()
+{
+	// trigger inputs
+	// tle8888 VR conditioner
+	boardConfiguration->triggerInputPins[0] = GPIOC_6;
+	boardConfiguration->triggerInputPins[1] = GPIO_UNASSIGNED;
+	boardConfiguration->triggerInputPins[2] = GPIO_UNASSIGNED;
+	// Direct hall-only cam input
+	engineConfiguration->camInputs[0] = GPIOA_5;
+
+	// tps
+	engineConfiguration->tps1_1AdcChannel = EFI_ADC_13;
+	engineConfiguration->tps2_1AdcChannel = EFI_ADC_NONE;
+
+	// clt = AN temp 1
+	engineConfiguration->clt.adcChannel = EFI_ADC_0;
+	engineConfiguration->clt.config.bias_resistor = 2700;
+
+	// iat = AN temp 2
+	engineConfiguration->iat.adcChannel = EFI_ADC_1;
+	engineConfiguration->iat.config.bias_resistor = 2700;
+}
+
+void setPinConfigurationOverrides(void) {
+}
+
+void setSerialConfigurationOverrides(void) {
+	boardConfiguration->useSerialPort = false;
+	engineConfiguration->binarySerialTxPin = GPIO_UNASSIGNED;
+	engineConfiguration->binarySerialRxPin = GPIO_UNASSIGNED;
+	engineConfiguration->consoleSerialTxPin = GPIO_UNASSIGNED;
+	engineConfiguration->consoleSerialRxPin = GPIO_UNASSIGNED;
+}
+
+
+/**
+ * @brief   Board-specific configuration code overrides.
+ * @todo    Add your board-specific code, if any.
+ */
+void setBoardConfigurationOverrides(void) {
+	engineConfiguration->baroSensor.hwChannel = EFI_ADC_NONE;
+	engineConfiguration->throttlePedalPositionAdcChannel = EFI_ADC_NONE;
+
+	// not used
+	engineConfiguration->dizzySparkOutputPin = GPIO_UNASSIGNED;
+	engineConfiguration->externalKnockSenseAdc = EFI_ADC_NONE;
+	engineConfiguration->displayMode = DM_NONE;
+	boardConfiguration->HD44780_rs = GPIO_UNASSIGNED;
```

Comment:
Do we need any of GPIO_UNASSIGNED in master? is not master NOT applying Frankenso configs? shall we consider the fact that we have all zeros and GPIO_UNASSIGNED is zero?